app/docs/dev.md

```diff
@@ -1,29 +1,207 @@
-# YourFunctionName
-*Brief description of what this evaluation function does, from the developer perspective*
+# buckinghamPiTheorem
+
+Checks that the set of quantities in the response matches the set of quantities in the sense given by the Buckingham Pi theorem.
+
+There are three different ways of supplying this function with the necessary information.
+1. In the answer, provide an example set of groups as a comma separated list. When used this way the function assumes that the given list is correct and contains at least the minimum number of groups.
```

Comment:
Suggest:
-  replace "answer" with "`answer`" (in back-ticks)
-  insert "valid" between 'example' and 'set'

Reply:
Changed according to suggestions